src/core/utils/misc.cpp

```diff
@@ -241,6 +242,11 @@ int Utils::Misc::pythonVersion()
         python_proc.start("python3", QStringList() << "--version", QIODevice::ReadOnly);
         if (python_proc.waitForFinished()) {
             if (python_proc.exitCode() == 0) {
+                QByteArray output = python_proc.readAllStandardOutput();
+                if (output.isEmpty())
+                    output = python_proc.readAllStandardError();
+                const QByteArray version_str = output.split(' ').last();
+                Logger::instance()->addMessage(QCoreApplication::translate("misc", "Python version: %1").arg(QString(version_str)), Log::INFO);
                 version = 3;
```

Comment:
This code was introduced by @pmzqla (see previous merged PR).
And it is enabled only for non-Windows OSes.